.github/workflows/pytest.yml

```diff
@@ -8,6 +8,7 @@ on:
     types: [opened, synchronize, reopened]
 env:
   MIN_COVERAGE_REQUIRED: 70
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
don't put the token in the env for the whole action.
Only add it to the env of the step that uploads to coveralls. We only need it here.